src/node_options.cc

```diff
@@ -149,6 +149,15 @@ void EnvironmentOptions::CheckOptions(std::vector<std::string>* errors) {
   }
 
 #if HAVE_INSPECTOR
+  if (!cpu_prof && !cpu_prof_name.empty()) {
+    errors->push_back("--cpu-prof-name must be used with --cpu-prof");
+  }
+
+  // We may add support for other profiles in the future.
```

Comment:
@mscdex See this line here, the intention is that --prof-dir can be used for other profiles in the future.

I believe it is possible to also write --prof output to a specified directory, it's just that v8 does not provide such an option at the moment, but we could work towards that. This flag is experimental anyway.

Reply:
Maybe a more obvious name would be `--node-prof-dir`?